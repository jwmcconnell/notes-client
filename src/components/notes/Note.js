import React from 'react';
import PropTypes from 'prop-types';

const Note = ({ title, body }) => {
  return (
    <li>
      <h6>{title}</h6>
      <p>{body}</p>
    </li>
  );
};

Note.propTypes = {
  title: PropTypes.string.isRequired,
  body: PropTypes.string.isRequired
};

export default Note;
