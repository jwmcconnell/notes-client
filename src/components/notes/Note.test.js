import React from 'react';
import { shallow } from 'enzyme';
import Note from './Note';

describe('Note component', () => {
  it('renders Note', () => {
    const wrapper = shallow(<Note 
      title="hi" 
      body="there" 
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
