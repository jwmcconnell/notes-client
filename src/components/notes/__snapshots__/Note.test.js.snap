// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders Note 1`] = `
<li>
  <h6>
    hi
  </h6>
  <p>
    there
  </p>
</li>
`;
